### PR TITLE
create npm build script

### LINKED ISSUE #23: use npm scripts and local installation of tools (like Gulp)
see https://github.com/kenzanlabs/keystone/issues/36

### PR DESCRIPTION
**Pull request by thescientist13**
resolves #23

### CODE REVIEW

**Comment by thescientist13** [comment]
Can one of the admins verify this patch?

**Comment by thescientist13** [comment]
ok to test

File: gulpfile.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ gulp.task('validateCSS', ['validateJS'], function() {
     .pipe(validateCSSPipeline.validateCSS());
 });
 
-gulp.task('default', ['validateJS', 'validateCSS'], function() {
+gulp.task('build', ['validateJS', 'validateCSS'], function() {
```
Comment (paul-barry-kenzan) [inline gulpfile.js]: Nice catch and update.

**Comment by thescientist13** [comment]
ok to test

**Comment by thescientist13** [comment]
ok to test

**Comment by thescientist13** [comment]
ok to test

**Pull request merged**